### PR TITLE
Show which URLs are being actively downloaded

### PR DESCRIPTION
**Pull request by markej**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 6094613d069abe5866ff7d9a0a862fd81c2fc3e6 available at: https://gist.github.com/c3cbc2efef84ba437931<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7366.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 7 - Looking at hifi://sandbox I can see download refs flash by and can see where this would be a good tool to see if something is hanging up/  QA Pass

File: interface/src/ui/Stats.cpp [hunk 1]
```diff
@@ -192,9 +192,29 @@ void Stats::updateStats(bool force) {
...
-        STAT_UPDATE(downloads, ResourceCache::getLoadingRequests().size());
+        QList<Resource*> loadingRequests = ResourceCache::getLoadingRequests();
+        STAT_UPDATE(downloads, loadingRequests.size());
         STAT_UPDATE(downloadLimit, ResourceCache::getRequestLimit())
         STAT_UPDATE(downloadsPending, ResourceCache::getPendingRequestCount());
+
+        // See if the active download urls have changed
+        bool updateUrls = _downloads != _downloadUrls.size();
+        if (!updateUrls) {
+            for (int ii = 0; ii < _downloads; ii++) {
```
Comment (ZappoMan) [inline interface/src/ui/Stats.cpp]: coding standard says to use single letter variable names for iterators like this.

**Comment by ZappoMan** [comment]
@OmegaHeron - can we verify that this doesn't have a negative impact on frame rate?

**Comment by OmegaHeron** [comment]
I saw no change to FPS outside typical variances I see in run sessions -- typically in a complex scene I'll see, say, 45 fps one run - 42 another.  This PR was similar in that - if there's an FPS hit it's too close to that variance to quantify for me.

I tried to convince myself that this PR was more subject to stuttering in movement while scene loads than production, but, after repeatedly trying both I couldn't quantify it as anything more than minuscule, if at all.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 66f9effd124eb1daeb8715b0f6bfd2ea5a9f520c available at: https://gist.github.com/c3cbc2efef84ba437931<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7366.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7366.dmg

**Pull request merged**